src/Progress/Progress.tsx

```diff
@@ -0,0 +1,84 @@
+import React, {useEffect, useRef, useState, forwardRef, memo} from 'react'
+import {View, StyleSheet, Animated, Dimensions} from 'react-native'
+
+interface IProgressProps {
+  /**
+   * Current percent of the Progress
+   * default 0, max 100
+   */
+  value: number
+  /**
+   * Defines height of Progress
+   * default 16
+   */
+  size?: number
+  /**
+   * Defines borderRadius of Progress
+   * default 0
+   */
+  borderRadius?: number
+  /**
+   * Defines color of Track Bar
+   * default #49BE25
+   */
+  filledTrackColor?: string
+  /**
+   * Defines background color of Progress
+   * default #E5E5E5
+   */
+  backgroundColor?: string
+}
+
+const {width: screenWidth} = Dimensions.get('screen')
```

Comment:
i think we should create and export screenWidth and screenHeight from utils folder

Reply:
Yes. will update when merge with other branch

Reply:
updated